### PR TITLE
Update GPSLogger URL to F-Droid

### PR DESCRIPTION
**Pull request by mendhak**
GPSLogger isn't on the Play Store, it's moved to F-Droid, so here I am updating the URL.

### CODE REVIEW

**Comment by michael63-osm** [comment]
Thank you for notifying us about this change. I had to perform the merge manually because we do not create translated pages within GitHub but use Transifex for translation.

My workflow thus looked like: first amend the English source on our staging site, push that new version to Transifex, make the changes to the url in all affected languages on Transifex, wait until it was automatically pulled by our staging site and finally transfer everything to the production site. Is less complicated than it sounds but took a while and now the production site 

Bottom line: we welcome amendments and corrections to our contents but as is explained in the readme file at the root of the repository it is totally sufficient to change English contents. Other languages are handled on Transifex.
